### PR TITLE
Add 'Höffner Online GmbH & Co. KG' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22hoeffner%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22commerce%22%5D%2C%22name%22%3A%22H%C3%B6ffner%20Online%20GmbH%20%26%20Co.%20KG%22%2C%22address%22%3A%22Am%20Rondell%201%5Cn12529%20Sch%C3%B6nefeld%22%2C%22phone%22%3A%22%2B49%20(0)30%2037444%20-02%20%20und%20%2B49%20(0)39205%2061-333%20%22%2C%22fax%22%3A%22%2B49%2030%20374448700%22%2C%22email%22%3A%22%20onlineservice%40hoeffner.de%22%2C%22web%22%3A%22https%3A%2F%2Fwww.hoeffner.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.hoeffner.de%2Fimpressum%22%2C%22https%3A%2F%2Fwww.hoeffner.de%2Fdatenschutz%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1297%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/hoeffner.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "slug": "hoeffner",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Höffner Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld",
+    "phone": "+49 (0)30 37444 -02  und +49 (0)39205 61-333 ",
```
Comment (moriarty-jim) [inline companies/hoeffner.json:11]:
```suggestion
    "phone": "+49 30 3744402",
```
I would suggest to use only the first number, the second one is for "online customer service"

File: companies/hoeffner.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "hoeffner",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Höffner Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld",
```
Comment (moriarty-jim) [inline companies/hoeffner.json:10]:
```suggestion
    "address": "Am Rondell 1\n12529 Schönefeld\nDeutschland",
```
fixed address

File: companies/hoeffner.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "phone": "+49 (0)30 37444 -02  und +49 (0)39205 61-333 ",
+    "fax": "+49 30 374448700",
+    "email": " onlineservice@hoeffner.de",
```
Comment (moriarty-jim) [inline companies/hoeffner.json:13]:
```suggestion
    "email": "datenschutz@hoeffner.de",
```
specific email-address for DPO

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
some fixes/suggestions

**Review by mal-tee — APPROVED** [review]
Thanks to both of you!

**Pull request merged**